Build: add ourTeam

ceyhungulbas opened the pull request:
closed #18  

![resim](https://user-images.githubusercontent.com/44509662/129962307-834b4a7a-5bd1-4a46-a1f4-9a12d1c6c896.png)

- Used map for rendering
- OurTeamForSmallScreen.jsx looks a little bit complicated. Lemme explain: there was a problem about creating 2 row and 2 column so i thought in create two person for this code block thats why i used ternary operator with `return index < 2 ?` in line 13 then applied same method for other two person with using `return 1 < index ?` 
- Used Card component from react-bootstrap
- yarn run fix
- Created snapshot with message "ourTeam rendered correctly"
- Used global hooks that called "windowSize.js" before this component -OurTeam- I created for Navbar

ceyhungulbas committed:
Build: add ourTeam

ceyhungulbas committed:
Build: ourTeam

ceyhungulbas committed:
Build: add ourTeam

ceyhungulbas committed:
Build: add ourTeam

ceyhungulbas committed:
Build: add ourTeam

udenizdemirbilek commented on src/App.js:
```diff
@@ -3,11 +3,13 @@ import { Container } from "react-bootstrap";
   return (
     <>
       <Container>
+        <OurTeam />
```
Isn't OurTeam component a part of AboutPage container?
![image](https://user-images.githubusercontent.com/81304552/130050029-9de29cc4-7356-4789-88cb-f2b0699f50f6.png)

ceyhungulbas replied:
Yes, it is! When Halit accept PR of About, i'm going to `git merge dev` on ourTeam branch then i'll add ourTeam component.

Edit:
Whe he assigned to me these components he said: Create two different components

udenizdemirbilek reviewed (changes requested):
Good work Ceyhun!
Found some minor issues and I've commented them.
I was wondering if OurTeam wouldn't be easier with bootstrap grid system? But apparently it works like this, so no problem.

ceyhungulbas committed:
Fix: solve conflicts

ceyhungulbas committed:
Fix: missing space

ceyhungulbas committed:
Build: add about

ceyhungulbas commented:
![resim](https://user-images.githubusercontent.com/44509662/130286306-cbc10f19-451e-4dd5-b08a-0a141a8b61be.png)


@udenizdemirbilek @halitbatur  PTAL

- Added Our Team in About page.
- Solved responsive problems
- Solved margin problem
- Updated tests
- In About.jsx solved missing empty space

udenizdemirbilek reviewed (approved):
Well done Ceyhun! Looks good to me

halitbatur commented on src/components/About/About.jsx:
```diff
@@ -16,7 +16,7 @@ const About = () => {
             <span className="spanDescription">
               <br />
-              Our Mision:
+              Our Mision:{" "}
```
remove th extra space

ceyhungulbas replied:
![resim](https://user-images.githubusercontent.com/44509662/130319354-4b24b214-81c9-4dd4-ba90-605fe4f346e4.png)
![resim](https://user-images.githubusercontent.com/44509662/130319361-1e07ceb6-4d5b-45a4-98fd-9d4404e9d918.png)
@halitbatur Thats why I added space, if you want I can remove it

halitbatur commented on src/components/OurTeam/OurTeamForSmallScreen.jsx:
```diff
@@ -0,0 +1,62 @@
+      <div className="teamMembersCardMainDiv">
+        <div className="teamMembersCard">
+          {ourTeamData.map((info, index) => {
+            return index < 2 ? (
```
can we use filter instead of map here since u are not using all the items only the ones with index less than 2 i think

ceyhungulbas replied:
wow great idea, i totally forgot about filter func. I'm on it

halitbatur commented on src/components/OurTeam/OurTeamForSmallScreen.jsx:
```diff
@@ -0,0 +1,62 @@
+        </div>
+        <div className="teamMembersCard">
+          {ourTeamData.map((info, index) => {
+            return 1 < index ? (
```
same here

ceyhungulbas committed:
Fix: add filter on map func

ceyhungulbas commented:
@halitbatur  PTAL

- Added filter function on map function
- Updated test
- Didnt delete empty space because of [this](https://github.com/ReCoded-Org/capstone-turkey-giftify/pull/30#discussion_r693338935)

halitbatur commented on src/components/OurTeam/OurTeamForSmallScreen.jsx:
```diff
@@ -0,0 +1,103 @@
+
+export default OurTeamForSmallScreen;
+
+// {ourTeamData.map((info, index) => {
```
no commented code please

ceyhungulbas committed:
Fix: remove commented code

ceyhungulbas commented:
@halitbatur  PTAL 

It was just backup, totally forget this code block
